modules/promotion/src/Annotation/PromotionOffer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\commerce_promotion\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines the promotion offer plugin annotation object.
+ *
+ * Plugin namespace: Plugin\Commerce\PromotionOffer.
+ *
+ * @Annotation
+ */
+class PromotionOffer extends Plugin {
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * An array of context definitions describing the context used by the plugin.
+   *
+   * The array is keyed by context names.
+   *
+   * @var \Drupal\Core\Annotation\ContextDefinition[]
+   */
+  public $context = [];
+
+  /**
+   * The target entity type this action applies to.
+   *
+   * For example, this should be 'commerce_order' or 'commerce_line_item'.
+   *
+   * @var string
+   */
+  public $targetEntityType;
```

Comment:
Annotation keys are snake_case, not camelCase. So it should be target_entity_type